hack/deploy-scheduler-estimator.sh

```diff
@@ -20,11 +20,11 @@ set -o nounset
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 function usage() {
   echo "This script will deploy karmada-scheduler-estimator of a cluster."
-  echo "Usage: hack/deploy-scheduler-estimator.sh <HOST_CLUSTER_KUBECONFIG> <HOST_CLUSTER_NAME> <MEMBER_CLUSTER_KUBECONFIG> <MEMBER_CLUSTER_NAME>"
+  echo "Usage: hack/deploy-scheduler-estimator.sh <HOST_CLUSTER_KUBECONFIG> <HOST_CLUSTER_NAME> <MEMBER_CLUSTER_KUBECONFIG> <MEMBER_CLUSTER_CONTEXT> [<MEMBER_CLUSTER_NAME> <SCHEDULER-ESTIMATOR-NAMESPACE> <GRPC_AUTH_SECRET_NAME>]"
```

Comment:
this is not requred, right? so how about pinrt two usage:

```shell
  echo "Usage: hack/deploy-scheduler-estimator.sh <HOST_CLUSTER_KUBECONFIG> <HOST_CLUSTER_NAME> <MEMBER_CLUSTER_KUBECONFIG> <MEMBER_CLUSTER_NAME>"
  echo "Usage: hack/deploy-scheduler-estimator.sh <HOST_CLUSTER_KUBECONFIG> <HOST_CLUSTER_NAME> <MEMBER_CLUSTER_KUBECONFIG> <MEMBER_CLUSTER_CONTEXT> [<MEMBER_CLUSTER_NAME> <SCHEDULER-ESTIMATOR-NAMESPACE> <GRPC_AUTH_SECRET_NAME>]"
```

Reply:
nice suggestion

Reply:
```shell
MEMBER_CLUSTER_CONTEXT=$4
MEMBER_CLUSTER_NAME=${5:-$4}
```

By default, `MEMBER_CLUSTER_CONTEXT` and `MEMBER_CLUSTER_NAME` are the same value.

Reply:
I'm fine with it. Previously, it was to maintain consistent behavior in the script. If there's no requirement for this, I'd be happy to modify it according to your suggestion.